packages/medusa/src/api-v2/admin/products/query-config.ts

```diff
@@ -22,6 +22,11 @@ export const defaultAdminProductsVariantFields = [
   "ean",
   "upc",
   "barcode",
+  "prices.id",
+  "prices.currency_code",
+  "prices.amount",
+  "prices.created_at",
+  "prices.updated_at",
```

Comment:
we need to append rules here as well and in the response, normalize it to a flexible object like:
```
{
  region_id: "region-id"
  sales_channel_id: "sales-channel-id"
}
```

Reply:
See my comment above, let's see how we should handle these first.